ts/esbuild/upstream.bzl

```diff
@@ -66,6 +66,7 @@ def _esbuild_impl(ctx):
     args.add_joined(["--platform", ctx.attr.platform], join_with = "=")
     args.add_joined(["--target", ctx.attr.target], join_with = "=")
     args.add_joined(["--log-level", "info"], join_with = "=")
+    args.add_joined(["--loader", ".svg=text"], join_with = ":")
```

Comment:
Just a note for the future: https://github.com/ankitects/anki/commit/f1780e6e24c66335ca9cb2ceb0d730d383d1a3a3